arduino/smartcar_mqtt/smartcar_mqtt.ino

```diff
@@ -44,14 +50,25 @@ void setup() {
 }
 
 void loop() {
+  static unsigned long previousFrame = 0;
   if (mqtt.connected()) {
     mqtt.loop();
+#ifdef __SMCE__
+    const auto currentTime = millis();
+    if(currentTime - previousFrame >= 65) {
+      previousFrame = currentTime;
+      Camera.readFrame(frameBuffer.data());
+      mqtt.publish("/smartcar/camera", frameBuffer.data(), frameBuffer.size(), false, 0);
+    }
+#endif
   }
+
   static unsigned long previousTransmission = 0;
   const auto currentTime = millis();
   if (currentTime - previousTransmission >= oneSecond) {
     previousTransmission = currentTime;
     const auto distance = String(front.getDistance());
     mqtt.publish("/smartcar/ultrasound/front", distance);
   }
```

Comment:
Thinking of it, these should be done (or rather the distance published), only if `mqtt` is connected.